Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -32,6 +32,8 @@ public LimbPos(LimbType limbType, Vector2 position, bool allowUsingLimb)
 
     partial class Controller : ItemComponent, IServerSerializable
     {
+        protected string output, falseOutput;
```

Comment:
These could be private

Reply:
Fixed in [af157a7](https://github.com/Regalis11/Barotrauma/pull/10419/commits/af157a79551d2add3ae4bbc5050212d39b18e77d).